packages/framework-stats/src/Packagist/Purifier/PHPStanNettePackagesPurifier.php

```diff
@@ -34,7 +34,7 @@ public function correctLastYearDownloads(int $yearDownloads, string $packageName
 
         // packagist doesn't provide way to count this number, so it is estimation compared to another nette packages,
         // and phpstan downloads from https://packagist.org/packages/phpstan/phpstan/stats
-        $phpstanLastYearInfluence = 5_500_000;
+        $phpstanLastYearInfluence = 3_500_000;
```

Comment:
The PHPStan 0.12 is not using Nette packages in installs anymore, just once on deploy from https://github.com/phpstan/phpstan-src